lib/constants.ts

```diff
@@ -14,5 +14,17 @@
  * limitations under the License.
  */
 
- export const LAMBDA_LAYER_ARN = "arn:aws:serverlessrepo:us-east-1:903779448426:applications/lambda-layer-kubectl"; 
-export const LAMBDA_LAYER_VERSION = "2.0.0-beta3";
\ No newline at end of file
+import { Duration } from "@aws-cdk/core";
+
+export const LAMBDA_LAYER_ARN = "arn:aws:serverlessrepo:us-east-1:903779448426:applications/lambda-layer-kubectl"; 
```

Comment:
Not totally related to this specific PR, but would it be better to extract ARN references out of Github, or perhaps parameterise it to not be hard-coded to a region/account?

Reply:
I was going by what's documented here: https://github.com/aws-samples/aws-lambda-layer-kubectl#deploy-with-aws-cdk
It looks like something similar is implemented in the [CDK Itself](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-eks/lib/kubectl-layer.ts)

Might be worth investigating?